workers/fogbow_data.py

```diff
@@ -0,0 +1,15 @@
+ras_url='https://services-atm-test-site2.lsd.ufcg.edu.br/ras'
+ras_public_key_endpoint=ras_url + '/publicKey'
+ras_public_ip_endpoint=ras_url + '/publicIps'
+ras_compute_endpoint=ras_url + '/computes'
+
+ras_member_id=
+ras_cloud_name=
+ras_images=ras_url + '/images/' + ras_member_id + '/' + ras_cloud_name 
+
+as_url='https://services-atm-test-site2.lsd.ufcg.edu.br/as'
```

Comment:
remove the https://services-atm-test-site2.lsd.ufcg.edu.br from here. it is specify to us not to the code

Reply:
Solved in d019b31ab3917488c031b30577ea43706a2dc636.